deploy_and_run_tests.sh

```diff
@@ -33,13 +35,15 @@ KEEP_IMAGE_OPT="--keep_image"
 IMAGE_TAG_OPT="--image_tag"
 PROJECT_OPT="--project"
 RUN_UNIT_TEST_OPT="--run_unit_tests"
+RUN_PREPROCESSOR_TEST_OPT="--run_preprocessor_tests"
```

Comment:
I think we need this to be triggered on each commit to the master GitHub branch as well, correct? Then you should add this to `cloudbuild_CI.yaml` as well.

Reply:
@arostamianfar and I have discussed about it. We decided to to disable it by default to make the integration tests more time efficient since once this project is done, the further commits may not touch the preprocessor much. Now I have some concerns since there are `common` scripts shared by both.